onnxruntime/contrib_ops/cpu/bert/attention_cpu_base.h

```diff
@@ -144,6 +152,23 @@ class AttentionCPUBase : public AttentionBase {
       });
     }
 
+    if (extra_add_qk_data != nullptr) {
```

Comment:
Maybe this part could be done in attention_probs initializations: copy or reuse the memory?

Reply:
I agree. 

Another option is to see whether math::Gemm<T, ThreadPool> accepts bias  parameter.

Reply:
after exploring few options I made the extra add in the parallel loop which probably should have been done in first place. Got a performance improvement of 15 ms with this. :)